script/main.py

```diff
@@ -19,10 +19,24 @@ def create_driver(name):
   return driver
 
 
+def list_of_all_links(driver):
+  links_elem = driver.find_elements_by_tag_name("a")
+  return links_elem  
+
+
+def print_list_of_all_links(driver):
+  links_elem = driver.find_elements_by_tag_name("a")
```

Comment:
Do not search for links here. It was already done.
Also the driver parameter is not necessary.